docs/generate-env.sh

```
@@ -0,0 +1,92 @@
+
+#!/usr/bin/env bash
```

Shellcheck (0.11.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)